parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetOutputFormat.java

```diff
@@ -341,7 +341,7 @@ public static void setMaxPaddingSize(Configuration conf, int maxPaddingSize) {
     conf.setInt(MAX_PADDING_BYTES, maxPaddingSize);
   }
 
-  private static int getMaxPaddingSize(Configuration conf) {
+  public static int getMaxPaddingSize(Configuration conf) {
```

Comment:
Changing method from private to public would need to add documentation and unit tests

Reply:
👍 I'll add it for the rest of the publics in the class as well while I'm at it.